requirements-dev.txt

```diff
@@ -2,5 +2,6 @@
 black
 mypy
 flake8==3.9.2
+jupyter
```

Comment:
@gabrieltseng Adding so developers always use the jupyter package within the virtual environment. This will add to the install time in ci but will not impact the package. Any thoughts?

Reply:
yup, this works for me

Reply:
I agree that makes sense